src/kes.rs

```diff
@@ -31,7 +32,7 @@ macro_rules! sum_kes {
 
         impl<'a> Drop for $name<'a> {
             fn drop(&mut self) {
-                self.0.copy_from_slice(&[0u8; Self::SIZE + 4])
+                self.0.zeroize();
```

Comment:
Why is that better? (I'd hope the commit message to explain that). 
May we add a simple unit test that demonstrate this has the expected effect?

Reply:
`copy_from_slice` does not guarantee that it will not be optimized away before running. `zeroize` employs volatile writes that ensures the code runs and is not optimized out.  `copy_from_slice` could also potentially allocate new memory, copy the zeroes into it and then leave the previous memory holding sensitive information.

Reply:
Fair justification! Shouldn't that be in the commit message body 🫣?

Reply:
A test may not be able to demonstrate the issue... namely, when copy_from_slice is optimized away, or causes a re-alloc to run before the copy happens.  In looking at the code, the original developer had already added the dependency on the zeroize crate. It's used in the upstream crypto libraries.  It looked to me like they just forgot to change this line of code.